src/app/lib/router.js

```diff
@@ -0,0 +1,44 @@
+import Router from 'ampersand-router';
+import HomePage from '../views/home_page';
+import ExperimentPage from '../views/experiment_page';
+import SettingsPage from '../views/settings_page';
+import NotFoundPage from '../views/not_found_page';
+
+const app = window.app;
```

Comment:
Ahh I see it's in the mix below... you can just import it with the other imports since it's a singleton and then you aren't dealing with globals...